app/viewmodels/govuk/RadiosFluency.scala

```diff
@@ -62,6 +63,38 @@ trait RadiosFluency {
         fieldset = FieldsetViewModel(legend)
       )
 
+    def yesNoItemsWithConditionalHtml(
+                                         field: Field,
```

Comment:
maybe we can reformat this to have the parameters aligned with the method? Meaning:
`field` right after the `(` above, and the rest of the parameters just aligned below like now